javascript/javascript3/week1/city.js

```diff
@@ -0,0 +1,62 @@
+
+const apiKey = "06150efd3d284c694f2952a08be1740b";
+const main = document.getElementById('main');
+const form = document.getElementById('form');
+const search = document.getElementById('search');
+const button = document.getElementById('myBtn');
+const url = (city) => `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${apiKey}`;
+async function getWeatherByLocation(city) {
+  const resp = await fetch(url(city), {
+    origin: "cros"
```

Comment:
It looks like `origin` is not a valid property of fetch options. It doesn't seem to do anything, you could safely remove it 🙂

Reply:
It's ok. But could you explain me how to get rid of "CROS platform issues" while using chrome browser. I had some issues while debugging in chrome console.

Reply:
Yeah, it's quite a complex topic. Basically, the API you're calling needs to support your current "domain" (like localhost or example.org) as a Cross-Origin. That requires some headers being sent from the API-side. Nothing you can immediately do about it, unfortunately, if you don't control the API. See https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS for more info